README.md

```diff
@@ -17,12 +17,5 @@ ALTLab's deployment server. Enables web application redeployment via HTTPS.
 * [How to register an application with the API](./registration.md)
 * [How to configure the servers](./docs/server-config.md)
 
-## Notes
-
-* The docker service should restart all running containers when restarted. See <https://stackoverflow.com/a/18797089/6626414>.
-* You can monitor the logs in `deploy.gunicorn`:
-  - `sudo journalctl -lfu deploy.gunicorn`
-  - `sudo journalctl -lu deploy.gunicorn --since='15m'` (last 15 minutes)
```

Comment:
If the unit name hasn’t changed and these monitoring commands still work and are relevant, we should leave them here in the README.

Reply:
The unit file name has changed, but the docs are still relevant. Updated!